### DISCUSSION TITLE
motor & battery specs

### DISCUSSION THREAD

**Discussion by putyn** [StuckAtPrototype/Racer#15] [General]
this looks like an awesome project!  I would really like to replicate it to play with my son.

Looking at the github project, there is no mention about the type of motors you are using, I'm assuming one of those coreless motors with a planetary gearbox, but what dimensions?
also at [4:33](https://www.youtube.com/watch?v=6jzG-BMannc&t=273s) you show a tray with batteries, can you recommend a place to get batteries?

**Comment by mr-ukas** [comment]
Hi, it's in the BOM, last rows : ) 
See:
https://github.com/StuckAtPrototype/Racer/blob/master/hardware/racer/racer-v0.2-BOM.csv

Battery	PKCELL LP552530	https://www.adafruit.com/product/4237
6mm Planetary DC 3V 1200RPM motors	https://www.aliexpress.us/item/3256801986204077.html

**Reply by StuckAtPrototype** [reply]
thank you!
